src/components/IOUConfirmationList.js

```diff
@@ -266,11 +267,11 @@ class IOUConfirmationList extends Component {
                         selectedOptions={this.getAllOptionsAsSelected()}
                     />
                     <Text style={[styles.p5, styles.textMicroBold, styles.colorHeading]}>
-                        {this.props.translate('iOUConfirmationList.whatsItFor')}
+                        {}
```

Comment:
🤔 Should we just get rid of this text component entirely?

Reply:
Yes, i missed it